content/en/blog/_posts/2017-09-00-Kubernetes-18-Security-Workloads-And.md

```diff
@@ -24,9 +24,7 @@ Transport Layer Security (TLS) [certificate rotation](/docs/admin/kubelet-tls-bo
 
 Kubernetes 1.8 promotes the core Workload APIs to beta with the apps/v1beta2 group and version. The beta contains the current version of Deployment, DaemonSet, ReplicaSet, and StatefulSet. The Workloads APIs provide a stable foundation for migrating existing workloads to Kubernetes as well as developing cloud native applications that target Kubernetes natively.  
 
-For those considering running Big Data workloads on Kubernetes, the Workloads API now enables [native Kubernetes support](https://apache-spark-on-k8s.github.io/userdocs/) in Apache Spark.  
-
-Batch workloads, such as nightly ETL jobs, will benefit from the graduation of [CronJobs](/docs/concepts/workloads/controllers/cron-jobs/) to beta.  
```

Comment:
Similarly here, as well.

Reply:
This is a blog post from 2017, we don't typically edit blog posts from that far back